app/models/training_module.rb

```diff
@@ -97,5 +97,17 @@ def slides
     end
   end
 
+  def translated(key)
```

Comment:
These are all mainly presentation-level concerns, so I think putting these in the presenter rather than the model would be a little nicer.